packages/types/src/sign-client/jsonrpc.ts

```diff
@@ -44,7 +44,6 @@ export declare namespace JsonRpcTypes {
       namespaces: SessionTypes.Namespaces;
       sessionProperties?: ProposalTypes.SessionProperties;
       sessionConfig?: SessionTypes.SessionConfig;
-      pairingTopic: string;
```

Comment:
Do we risk breaking any externally used public interfaces by removing this?

Reply:
`wc_sessionSettle` is internal request so technically shouldn't be used directly by devs in any way